### PR TITLE
[MI-2731]:Added additional Loggings for fixing issue #97

### PR DESCRIPTION
**Pull request by Kshitij-Katiyar**
**Summary** 

- Added additional loggings for the issue with Welcomebot that when a user selects an Action, Welcomebot returns the following error in the chat and nothing in the Mattermost logs:
```WelcomeBot Error: We could not decode the action```

**Issue** 

- [Issue #97](https://github.com/mattermost/mattermost-plugin-welcomebot/issues/97)

### CODE REVIEW

**Comment by hanzei** [comment]
@Kshitij-Katiyar Please merge `master` into your branch

File: server/http_hooks.go [hunk 1]
```diff
@@ -13,6 +13,7 @@ import (
 func (p *Plugin) ServeHTTP(c *plugin.Context, w http.ResponseWriter, r *http.Request) {
 	var action *Action
 	if err := json.NewDecoder(r.Body).Decode(&action); err != nil || action == nil {
+		p.API.LogError("failed to decode action from request body", "error", err)
```
Comment (hanzei) [inline server/http_hooks.go:16]: Let's lower this to debug. It's not an error on our side.

Comment (hanzei) [inline server/http_hooks.go:16]:
```suggestion
		p.API.LogError("failed to decode action from request body", "error", err.Error())
```

**Review by hanzei — CHANGES_REQUESTED** [review]

**Comment by Kshitij-Katiyar** [comment]
> @Kshitij-Katiyar Please merge `master` into your branch

@hanzei Merged master and fixed the review comments

**Review by hanzei — APPROVED** [review]
LGTM

**Pull request merged**